pax/datastructure.py

```diff
@@ -278,6 +278,7 @@ def contributing_channels(self):
     #: Number of channels that have a hit maximum within a short (configurable) window around the peak's sum
     #: waveform maximum.
     tight_coincidence = INT_NAN
+    tight_coincidence_thresholds = np.zeros(5, dtype=np.int16)  # various tight coincidence levels
```

Comment:
This should be `np.array([], dtype=np.int16)` otherwise the ROOT class doesn't know how to write the C++ schema.

Reply:
Look at other array fields and how they are set within plugins.

Reply:
https://github.com/XENON1T/pax/blob/6bc98d8fb41b600308aed44c423c8a682c6f7ee9/pax/datastructure.py#L300 ?